idevice/src/services/dvt/notifications.rs

```diff
@@ -0,0 +1,161 @@
+                                    for (key, value) in dict.iter() {
+                                        match key.as_str() {
+                                            "mach_absolute_time" => {
+                                                if let Value::Integer(time) = value {
+                                                    notification.mach_absolute_time =
+                                                        time.as_signed().unwrap_or(0);
+                                                }
+                                            }
+                                            "execName" => {
+                                                if let Value::String(name) = value {
+                                                    notification.exec_name = name.clone();
+                                                }
+                                            }
+                                            "appName" => {
+                                                if let Value::String(name) = value {
+                                                    notification.app_name = name.clone();
+                                                }
+                                            }
+                                            "pid" => {
+                                                if let Value::Integer(pid) = value {
+                                                    notification.pid =
+                                                        pid.as_unsigned().unwrap_or(0) as u32;
+                                                }
+                                            }
+                                            "state_description" => {
+                                                if let Value::String(desc) = value {
+                                                    notification.state_description = desc.clone();
+                                                }
+                                            }
+                                            _ => {
+                                                println!("Unknown key: {} = {:?}", key, value);
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                            Err(e) => {
+                                println!("Failed to decode archive: {:?}", e);
+                            }
+                        },
+                        _ => {
+                            println!("Non-array aux value: {:?}", v);
+                        }
+                    }
+                }
+            }
+
+            let data = match message.data {
+                Some(Value::String(data)) => Some(data),
+                _ => None,
+            };
+            println!("{:#?}", (data.unwrap(), notification));
```

Comment:
For a library API, printing isn't what we want. It should be up to user code to print (or not print) whatever data is returned by the API.